### PR TITLE
Galleryscreen

### PR DESCRIPTION
**Pull request by JephDiel**
Added gallery screen with default cards. Merged tag component code from master and the bug fix from tag-testing

### CODE REVIEW

**Comment by nchaloult** [comment]
I think this PR might be missing a file. When I pull this branch down & try to run w/ `$ expo start`, I get:

```
Unable to resolve module `../components/Spacer` from `src/screens/ArtworkScreen.tsx`
```

File: mobile-app/src/components/Card.tsx [hunk 1]
```diff
@@ -68,15 +69,21 @@ const styles = StyleSheet.create({
...
 // Component
 
-interface Props {
+export interface Props {
   artwork: {
     title: string;
     artist: string;
     imageURLs: string[];
   };
 }
```
Comment (nchaloult) [inline mobile-app/src/components/Card.tsx:78]: I'm not sure that this needs to be exported. Is this interface being used in another file?

**Review by nchaloult — CHANGES_REQUESTED** [review]
Looking forward to seeing how this looks on my device

File: mobile-app/src/components/Card.tsx (hunk 1 above)
Comment (JephDiel) [reply mobile-app/src/components/Card.tsx]: Thanks for finding those, should be all fixed now.

**Review by szambranao — APPROVED** [review]

**Pull request merged**